versioned_docs/version-8.7/components/modeler/web-modeler/web-modeler-settings.md

```diff
@@ -0,0 +1,70 @@
+---
+id: web-modeler-settings
+title: Web Modeler settings
+description: You can configure email notifications and process application deployment policies in the Web Modeler settings.
+---
+
+import WebModelerSettings from './img/web-modeler-settings/web-modeler-settings.png'
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+Navigate to the Web Modeler settings to configure email notifications and the process application deployment policy.
+You can find the settings by clicking on your user icon in the top right corner of the Web Modeler and selecting **Settings**.
+
+<img src={WebModelerSettings} alt="Web Modeler settings" />
+
+## Email notifications
+
+Configure the projects for which you will receive email notifications when a collaborator mentions you in a comment.
+
+## Process application deployment
+
+Organization admins can configure the deployment policy for process applications in the Web Modeler settings.
+
+<Tabs groupId="deployment-permissions" defaultValue="saas" queryString values={
+[
+{label: 'SaaS', value: 'saas' },
+{label: 'Self-Managed', value: 'self-managed' },
+]}>
+
+<TabItem value='saas'>
+
+By default, only organization administrators can deploy process applications to clusters marked as
```

Comment:
Perhaps we should link to documentation on how to be granted admin permissions given we mention this a few times in this doc. Something like [here](https://docs.camunda.io/docs/components/console/manage-organization/manage-users/#:~:text=Roles%20and%20permissions%E2%80%8B&text=The%20admin%20role%20has%20the,to%20Console%2C%20except%20deletion%20privileges).

Reply:
I linked to the respective page as we are using the org admin role.